edu.stanford.protege.json

```diff
@@ -10,7 +10,8 @@
         "--socket=x11",
         "--share=ipc",
         "--share=network",
-        "--filesystem=home"
+        "--filesystem=home",
+        "--persist=.Protege"
```

Comment:
This does nothing since we already allow access to the home filesystem. 

https://docs.flatpak.org/en/latest/sandbox-permissions.html

Reply:
This maps the directory, so settings are shared between a local installation and a Flatpak. I also had a feeling that I needed this for updates to work.

Reply:
Ah I see
> This only works if the application has no home or a broader permission like host that includes home.